Editor/Auditors/ScriptAuditor.cs

```diff
@@ -56,6 +67,7 @@ public IEnumerable<ProblemDescriptor> GetDescriptors()
         public IEnumerable<IssueLayout> GetLayouts()
         {
             yield return k_IssueLayout;
+            yield return k_GenericIssueLayout;
```

Comment:
Nitpick really, but in other parts of the code this is called "Generics" with an s. Might be nice to name it consistently.